### PR TITLE
magento/devdocs#1455: Document default/core cookies

### PR DESCRIPTION
**Pull request by jasonevans1**
## This PR is a:

- [X] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary
Added a document to have a table where each cookie set by the Magento 2 core application is documented with a short description. See issue magento/devdocs#1455

When this pull request is merged, it will add a cookie reference table to the devdocs.

## Additional information
This is a new document. I did not know where in the menu this document should be put?

I am not sure what the URL should be?

whatsnew
Added a [new reference topic](https://devdocs.magento.com/guides/v2.3/config-guide/cookie-reference.md) about cookies set by Magento 2.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3212) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3212) before we can accept your contribution.<br/>

File: guides/v2.3/config-guide/cookie-reference.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
```
Comment (jeff-matthews) [inline guides/v2.3/config-guide/cookie-reference.md]: You can add new topics to the site navigation by adding an entry to the TOC file

> _data/toc/configuration-guide.yml

**Comment by jeff-matthews** [comment]
Thanks for updating the table format @jasonevans1. While I was trying to figure out the best place to put this on the devdocs site, I realized that we already have a [Cookie Reference](https://docs.magento.com/m2/ce/user_guide/stores/cookie-reference.html) in the Admin User Guide. It may need to be updated, but I'm not sure if we need it in both devdocs and the Admin guide.

**Comment by jasonevans1** [comment]
I agree we probably don't need it in both places. It would be nice to have a link to the admin guide cookie reference in the dev docs. As a developer I don't look at the admin guide very often or when I am looking for technical info.

**Comment by jeff-matthews** [comment]
If you could put a link to the cookie reference in devdocs site, where would you put it?

**Comment by magento-cicd2** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**